Gemfile.lock

```diff
@@ -1029,4 +1037,4 @@ RUBY VERSION
    ruby 2.5.0p0
 
 BUNDLED WITH
-   1.16.1
+   1.16.2
```

Comment:
Is it expected that we're bumping the patch version? Probably not an issue, just double-checking.

Reply:
this was not intentional, so i'm not sure why the version was bumped. my local version is 1.16.2, so maybe when i added the new gem and bundled, it bumped the version?